src/tasks/TaskScheduler.ts

```diff
@@ -0,0 +1,7 @@
+import Task from './Task';
+
+export default class TaskScheduler {
+	public static add( task: Task, interval: number ): void {
+		setInterval( task.run, interval );
```

Comment:
Current issue with this implementation: The task is not canceled and the interval still continues even after the bot has been shutdown using `!jira shutdown`. Only relevant for me, but I plan to add a restart command to the bot at some point, which would make this even more messy.